README.md

```diff
@@ -127,9 +127,27 @@ class CustomKotlinSerialization : SerializationModule {
             else -> kotlinSerialization.serialize(value)
         }
 }
+```
+
+Take into account that the ``KotlinSerialization`` class uses ``Gson`` under the hood. This class transforms your instances into json strings you can easily review when needed. On top of the json serialization we add some metadata really useful when serializing sealed hierarchies or objects. If for some reason you need to extend the serializer and use your own custom serializer also based on Gson you can do it as follows:
```

Comment:
Yeah, this was my idea about custom the Json serialization, but if you do this you miss the default Json Implementation?

Reply:
What do you mean? You can always customize the serializer as explained in the documentation. I think you are talking about raw strings being passed as inputs.